server/utils/tasks/table.ts

```diff
@@ -19,7 +27,13 @@ const unallocatedTableRows = (tasks: Array<Task>): Array<TableRow> => {
   const rows = [] as Array<TableRow>
 
   tasks.forEach(task => {
-    rows.push([nameCell(task), daysUntilDueCell(task), statusCell(task), taskTypeCell(task)])
+    rows.push([
+      nameCell(task),
+      daysUntilDueCell(task),
+      statusCell(task),
+      taskTypeCell(task),
+      allocationLinkCell(task, 'Allocate'),
```

Comment:
Love this rows.push approach, makes things so much clearer!

Reply:
Agreed! 
Something to take forward to any other tables we do.